utils/constants/common-constants.ts

```diff
@@ -0,0 +1,33 @@
+import { HomeIcon, BookIcon, SettingIcon } from '../../assets/icons';
+
+interface SidebarItem {
+  id: number;
+  iconName: string;
+  path: string;
+  icon: () => JSX.Element;
+  position: number;
+}
+
+export const sidebarItems: SidebarItem[] = [
+  {
+    id: 1,
+    iconName: 'Home',
+    path: '/dashboard',
```

Comment:
Well, I would like to suggest you to define another constant which includes these paths and then use it here. For example:
```javascript
const PAGE_ROUTES = {
  Dashboard: '/dashboard',
  Leads: '/leads',
  Settings: '/settings'
};
```
Now use it like:
```javascript
path: PAGE_ROUTES.Dashboard,
```